lib/trento_web/openapi/v1/schema/host.ex

```diff
@@ -51,6 +58,7 @@ defmodule TrentoWeb.OpenApi.V1.Schema.Host do
           type: :string,
           description: "Version of the agent installed on the host"
         },
+        health: ResourceHealth,
```

Comment:
Nitpick.
Not about this field.
I have checked a bit the code, and the value we could deprecate is the `deregistered_at`.
It is not used in the frontend anywhere.
So we could remove it in a potential, v2 version.
Anyway, it is not strictly about this PR. Up to you. If you don't do it, it is not a big deal.